sql/core/src/test/resources/sql-tests/results/postgreSQL/float8.sql.out

```diff
@@ -319,37 +319,37 @@ struct<five:string,f1:double,round_f1:double>
 -- !query
 select ceil(f1) as ceil_f1 from float8_tbl f
 -- !query schema
-struct<ceil_f1:bigint>
+struct<ceil_f1:double>
 -- !query output
--34
-0
-1
-1005
-9223372036854775807
+-34.0
+0.0
+1.0
+1.2345678901234E200
+1005.0
```

Comment:
In order to avoid any breaking changes, shall we cast the output type of `ceil`/`floor` to Long if the scale is 0 ?